.travis.yml

```diff
@@ -24,7 +24,7 @@ env:
     - WHEELHOUSE="$HOME/.cache/wheelhouse"
     - LIBXML2_DEB="libxml2-dbg_2.9.1+dfsg1-3ubuntu4.9_amd64.deb"
     - POSTGRES_DEB="postgresql-9.6_9.6.3-1.pgdg12.4+1_amd64.deb"
-    - ELASTICSEARCH_ARCHIVE="elasticsearch-2.4.5.tar.gz"
+    - ELASTICSEARCH_ARCHIVE="elasticsearch-6.3.1.tar.gz"
```

Comment:
This seems like too big of a jump - I don't know if we can just jump from es2 to es6 - 
https://www.elastic.co/guide/en/elasticsearch/reference/6.0/setup-upgrade.html

Seeing things about needing to reindex before upgrading, I imagine this upgrade is going to be a lot more involved - if we have to upgrade, can we bump to a lower version? @mfraezz, do you have any insight?

Reply:
if it's just travis's ES then it might be fine, I just don't know the implications here.

Reply:
We are not actually changing our "real" version of ES here just the massively out of date one the travis uses. We are actually using 6.3.1 for search already.